doc/nb/compiled_example.cfg

```diff
@@ -1,53 +1,25 @@
-# PypIt execution options.
-[run]
-    # Number of CPUs to use (-1 means all bar one CPU, -2 means all bar
-    # two CPUs)
-    ncpus = 1
-    # Flag to skip the data reduction and only check that all
-    # calibration data are present
-    calcheck = False
-    # The window of time in hours to search for calibration frames for a
-    # science frame
-    calwin = 0
-    # If True, run in setup mode.  Useful to parse files when starting
-    # reduction on a large set of data
-    setup = False
-    # If True, run quality control in real time.  Otherwise, checks are
-    # saved to disk for later inspection
-    qa = False
-    # If True, pypit will prepare the calibration frames; if false, will
-    # only reduce the science frames
-    preponly = False
-    # If True, pypit will stop and require a user carriage return at
-    # every quality control check
-    stopcheck = False
-    # If True, file sorting will ensure that the idname is made
-    useIDname = False
-    # Level of screen output: 0 supresses all output; 1 provides high-
-    # level output; 2 provides all output
-    verbosity = 2
-    # Directory relative to calling directory to write master files.
-    caldir = MF
-    # Directory relative to calling directory to write science files.
-    scidir = Science
-    # Directory relative to calling directory to write qa files.
-    qadir = QA
-    # File for the details of the sorted files.  If None, no output is
-    # created
-    sortdir = lris_blue_long_400_3400_d560
-    # Flag to overwrite any existing output files
-    overwrite = True
-
 # PypIt reduction rules.
 [rdx]
```

Comment:
Do we want these .cfg files here (or at all)?

Just checking.

Reply:
I've left them in just as an example of how to use `PypitPar`.  We can get rid of them because I don't expect many of us will want to use these cfg files, and will use the pypit files instead.